selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/dependencies/TransitiveDependencyTest.java

```diff
@@ -50,6 +55,8 @@
   @Inject private Loader loader;
   @Inject private PopupDialogsBrowser popupDialogsBrowser;
   @Inject private TestProjectServiceClient testProjectServiceClient;
+  @Inject private ActionsFactory actionsFactory;
```

Comment:
Are we really need this field after moving create action code to the SeleniumwebDriverHelper?